### PR TITLE
Adapts documentation for Pekko

### PR DESCRIPTION
**Pull request by nvollmar**
There are still some more references/links to Lightbend/Akka that need to be re-worked.

We're also missing some sample projects:
* https://github.com/akka/akka-sample-cluster-kubernetes-java
* https://github.com/akka/akka-sample-cluster-kubernetes-dns-java

We might to incorporate them into https://github.com/apache/incubator-pekko-samples

### CODE REVIEW

**Comment by pjfanning** [comment]
With the 2 extra Akka repos: that code is not Business Software Licensed. If someone volunteers to produce Pekko equivalents, they could use their own non-Apache forks. Alternatively, we could create Pekko repos when a volunteer pops up.

For now, we could just document that those Akka-based samples are still useful for learning - that anything learned from them is still valid with Pekko (users just need to migrate their code to use Pekko libs instead).

File: docs/src/main/paradox/kubernetes-lease.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The API, configuration and behavior may change based on feedback from initial us
 
 @@@
 
-This module is an implementation of an [Akka Coordination Lease](https://doc.akka.io/docs/akka/current/coordination.html#lease) backed 
+This module is an implementation of a [Akka Coordination Lease](https://doc.akka.io/docs/akka/current/coordination.html#lease) backed
```
[the remark this answers is not in the document]
Comment (nvollmar) [reply docs/src/main/paradox/kubernetes-lease.md:10]: will fix that

File: docs/src/main/paradox/bootstrap/recipes.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ $ sbt shell
 
 You can run multiple different Pekko Bootstrap-based applications in the same namespace,
 alongside any other containers that belong to the same logical application.
-The resources in `kubernetes/pekko-cluster.yml` are configured to run in the `akka-bootstrap-demo-ns` namespace.
```
Comment (pjfanning) [inline docs/src/main/paradox/bootstrap/recipes.md:150]: I haven't tested #36 but I'm hoping CI build is enough to prove the new names are ok

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**